transcriber/mock-server/.env.sample

```
@@ -0,0 +1,4 @@
+MODAL_API_KEY=sk-supersecretkey
+MODAL_CALLBACK_SECRET=supersecret
+MODAL_WORKER_URL=http://modal-worker:8000
+NGROK_AUTH_TOKEN=ngrok-supersecret-token
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)